modules/acs-architecture-connections.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * architecture/acs-architecture.adoc
+:_mod-docs-content-type: CONCEPT
+[id="acs-architecture-connections_{context}"]
+= {product-title-short} connection protocols and default ports
+
+The protocols and default ports for connections within {product-title-short} and between {product-title-short} and external components are shown in the following table.
+
+.{product-title-short} connections between components
+
+[cols="3",options="header"]
+|===
+| Component or external entity | Connection type | Port
+
+| Central and external services, for example, Jira and PagerDuty
+| HTTPS
+| ?
+
+| Central and secured cluster
+| HTTPS
+| 443
+
+| Central and sensor
+| TCP
+| 443
+
+| Central and user ({product-title-short} web portal or REST API)
+| HTTPS
+| 443
+
+| Central and user (CLI)
+| gRPC
+| 443
+
+| Scanner and Scanner-DB
+| TCP
+| 5432
+
+| Scanner and vulnerability feeds
+| HTTPS
+| ?
+
+| Scanner and image registries
+| HTTPS
+| ?
+
+| Sensor and admission controller
+| gRPC
+| ?
+
+| Sensor and collector
+| gRPC
+| ?
+
+|===
+
+//old graphic version: https://docs.openshift.com/acs/3.70/architecture/acs-architecture.html#external-components_acs-architecture
```

Comment:
curious: why switch from graphic to tabular view?

Reply:
I wasn't involved in that decision but I think because it takes months to get graphics updated and in theory, a text update is quicker, if info changes.